pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/integrationLayer/SearchView.kt

```diff
@@ -230,6 +228,65 @@ private fun BuSelector(listBu: List<Bu>, selectedBu: Bu, onBuSelected: (Bu) -> U
     }
 }
 
+@Composable
+private fun SearchInput(
+    query: String,
+    modifier: Modifier = Modifier,
+    prefix: @Composable (() -> Unit)?,
```

Comment:
⚠️ **[Android Lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>ComposableLambdaParameterPosition: Composable lambda parameter should be the last parameter so it can be used as a trailing lambda